src/debug/logger.cpp

```diff
@@ -61,11 +61,13 @@ namespace dynamicgraph
        (m_lv==VERBOSITY_INFO_WARNING_ERROR && isDebugMsg(type)))
       return;
 
+    //TODO Modif ici
     // if print is allowed by current verbosity level
     if( isStreamMsg(type))
       {
+        int l = line;
         // check whether counter already exists
-        string id = file+toString(line);
+        string id = file+/*toString(line)*/+ l;
```

Comment:
I don't know if I really do that or it was just a bug when I was trying to compile the code

Reply:
Actually, I think this line does not compile. What do you want to do ?

Reply:
I think this line didn't compile anyway

Reply:
Now that I understand the code, your change add a bug. Could you change it so that the string id is the concatenation of `file` and `line`.
You can use `ostringstream` from `sstream` file for that.

Reply:
Like os << file << line << std::endl ?

Reply:
Maybe I just add line because it wasn't exist on the first version of the code I just pull
https://www.noelshack.com/2019-33-1-1565606546-annotation-2019-08-12-124205.jpg

Reply:
I don't think it works because you removed `toString` in 425ee7c0d1ccf85cdca11577ef51947ab337bbce
You can replace it by:
```cpp
#include <sstream>
...
std::ostringstream oss;
oss << file << line;
std::string id (oss.str());
```